plugins/trace-activation/trace/add_event_intercepter.go

```diff
@@ -0,0 +1,51 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package traceactivation
+
+import (
+	"github.com/apache/skywalking-go/plugins/core/operator"
+	"github.com/apache/skywalking-go/plugins/core/tracing"
+)
+
+type AddEventInterceptor struct {
+}
+
+func (h *AddEventInterceptor) BeforeInvoke(invocation operator.Invocation) error {
+	var (
+		defaultEventType  tracing.EventType = "info"
```

Comment:
Can we make this a const otherwise variable in function?

Reply:
Sure, I've thought about it. But I'm not sure if creating a new file to make it shareable is the right approach, I'd like to hear your opinion

Reply:
You could create a new const file to including these.

Reply:
Done